dbt/adapters/bigquery/retry.py

```diff
@@ -113,16 +117,3 @@ def on_error(error: Exception):
                 raise FailedToConnectError(str(e))
 
     return on_error
-
-
-def _is_retryable(error: Exception) -> bool:
```

Comment:
I moved this into the class that calls it as a static method to make it more clear that this isn't used anywhere else. In fact, we wouldn't need this at all if we remove `BadRequest` as retryable since `_is_retryable == _job_should_retry` in that scenario.